gulp/tasks/test_ci.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        configFile = 'karma.ci.conf.js';
+
+    gulp.task('test_ci', function (callback) {
```

Comment:
Is the idea that Travis is going to run more comprehensive tests than the developer will run locally? I wonder if that's a bad idea because then the developer will have a false sense of confidence when the tests pass. @AlasdairSwan what's your feeling about this?